### PR TITLE
task 2.14.3

### PR DESCRIPTION
**Pull request by tacya9**

### CODE REVIEW

File: 2.14/2.14.3/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.14.3
+Введите число от 0 до 9 текстом
+Вывести алерт с сообщение: "Вы ввели цифру Х"
```
Comment (github-mix) [inline 2.14/2.14.3/index.html:10]: X is number, not text

File: 2.14/2.14.3/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+<script>
+	const userNum = prompt('Введите число от 0 до 9 текстом');
+
+	switch (userNum) {
+		case 'ноль':
+			alert('Вы ввели цифру ноль');
```
Comment (github-mix) [inline 2.14/2.14.3/index.html:16]: in each case, you have the phrase which starts from 'Вы ввели цифру'. Can we make a separate variable with this value?
And then try to rewrite your code in order to have only one place with the function 'alert"

**Review by github-mix — APPROVED** [review]

File: 2.14/2.14.3/index.html (hunk 1 above)
Comment (github-mix) [inline 2.14/2.14.3/index.html:10]: .

**Review by github-mix — CHANGES_REQUESTED** [review]

File: 2.14/2.14.3/index.html (hunk 1 above)
Comment (tacya9) [reply 2.14/2.14.3/index.html:10]: add comments txt file

**Review by github-mix — APPROVED** [review]

**Pull request merged**